kubernetes/polkassembly/values-parity-prod.yaml

```diff
@@ -13,13 +13,20 @@ frontend:
   ingress:
     enabled: true
     annotations:
-      traefik.ingress.kubernetes.io/redirect-entry-point: https
+      kubernetes.io/ingress.class: traefik-external
+      traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
+      cert-manager.io/cluster-issuer: letsencrypt-http01
+      traefik.ingress.kubernetes.io/router.tls: "true"
     hosts:
       - host: kusama.polkassembly.io
         path: /
       - host: polkassembly.io
         path: /
-    tls: []
+    tls:
+      - hosts:
+        - polkassembly.io
+        - kusama.polkassembly.io
+        secretName: polkassembly.io
```

Comment:
is this the right secret name?

Reply:
yes! the secret does not exist now. once the PR is merged, the certificate will be store in that secret and could be used in the ingress.